WordPress/build.gradle

```diff
@@ -238,6 +245,24 @@ android {
     }
 }
 
+def addBuildConfigFieldsFromPrefixedProperties(variant, properties, prefix) {
+    println("Adding properties prefixed with '${prefix}.' to ${variant.name} product flavor...")
+    // e.g. for prefix='wp', add properties named "wp.xxx" to our BuildConfig
+    properties.any { property ->
```

Comment:
This comes from the original code below, but I just realized something: shouldn't we use `each` rather than `any`?

Not yet fully familiar with the DSL, but to me, `any` sounds like a "filter" operation, testing if _any_ of the items in an array matches the predicate given by the closure/block and returning `true` if at least one item returned true for that predicate. So feels like we're using this `any` for its side effect rather than its original intent?
And we might have a similar incorrect use for `.all` elsewhere in our `build.gradle` scripts where we might need to replace them with `each` instead too?